jest.config.js

```diff
@@ -1,5 +1,11 @@
+// Removes the warning that we are using an unsupported version of TypeScript.
+process.env.TS_JEST_DISABLE_VER_CHECKER = false;
```

Comment:
Oh does this needs to be `true`? 🤔

Reply:
Yeah, you're right 😅 

https://kulshekhar.github.io/ts-jest/docs/version-checking/

Reply:
Done.